src/com/griddynamics/cinema/CinemaCli.java

```diff
@@ -0,0 +1,134 @@
+package com.griddynamics.cinema;
+
+import java.util.Scanner;
+
+public class CinemaCli {
+    
+    private static final int ONE_HUNDRED_PERCENT = 100;
+    
+    private static final int LARGE_CINEMA_HALL_MINIMUM_SEATS_AMOUNT = 60;
+    private static final int TICKET_FULL_PRICE = 10;
+    private static final int TICKET_DISCOUNT_PRICE = 8;
+    
+    private final Scanner scanner = new Scanner(System.in);
+    
+    private final boolean isLargeCinemaHall;
+    
+    private final int amountOfRows;
+    private final int amountOfSeatsInEachRow;
+    private final int totalSeatsAmount;
+    
+    private final int amountOfRowsWithFullPriceSeats;
+    private final int amountOfRowsWithDiscountPriceSeats;
+    
+    private final Cinema cinema;
+    
+    public CinemaCli(final Cinema cinema) {
```

Comment:
About the naming, sorry for misleading a bit, if you have cinema as input for this class, then it should not call `Cli`.
This object operates cinema. Let's call it CinemaOperator, what do you think? Feel free to suggest anything else

Reply:
It's OK. I will change the name according to your advice.